cluster-scope/base/user.openshift.io/groups/PrometheusAI/group.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: user.openshift.io/v1
+kind: Group
+metadata:
+  name: PrometheusAI
```

Comment:
@HumairAK should the name be all lowercase?

Reply:
yep, [ref](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)